src/engine/enginechannel.h

```diff
@@ -39,7 +39,8 @@ class EngineChannel : public EngineObject {
     };
 
     EngineChannel(const ChannelHandleAndGroup& handle_group,
-                  ChannelOrientation defaultOrientation = CENTER);
+                  ChannelOrientation defaultOrientation = CENTER,
+                  bool isInput = false);
```

Comment:
It is a technical requirement to distinguish them, because one goes to the talkover mix, which is added to the master at the very last and the other is treated as a normal deck input. 
Only if you mix in the signal to master only and not to booth and headphones the latency compensation can work.